Privacy Policy.md

```diff
@@ -1,23 +1,28 @@
+_Note: Please be aware that only our Terms of Service and Privacy Policy in English are legally binding. While translations of these documents may be provided for your convenience, they are translated by the community and are thus not legal documents._
 
 This Privacy Policy applies to the official instance of Revolt, available at revolt.chat. Self-hosted instances may choose to adopt this Privacy Policy if they wish.
 
 ### Introduction
 
-We get that legal documents can be overwhelming and boring, so we've tried our best to keep it as simple and possible while providing the information needed. It is important however to remember that these terms are legally binding between yourself and us if you choose to use Revolt, so please read them carefully before proceeding - We will continue to improve and tackle some sections. 
+We get that legal documents can be overwhelming and boring, so we've tried our best to keep it as simple and possible while providing the information needed. It is important however to remember that these terms are legally binding between yourself and us if you choose to use Revolt, so please read them carefully before proceeding - We will continue to improve and tackle some sections.
 
 ### Your Privacy
 
 Your privacy is very important to us, and we will ensure protecting your privacy when using our services.
 
-**Revolt** is an open-source project (source code is available on our [GitLab](https://gitlab.insrt.uk/revolt)), and we respect our user's privacy while remaining transparent with our users. We only collect limited data from our users that is absolutely necessary.
+**Revolt** is an open-source project (source code is available on our [GitHub](https://github.com/revoltchat)), and we respect our user's privacy while remaining transparent with our users. We only collect limited data from our users that is absolutely necessary.
 
 - **Account Information**: We collect data from our users when they register for an account on Revolt, such as: the registered e-mail address, hashed password, and chosen username.
 - **Servers and Groups**: If you create or join a group/server on Revolt, we collect and store the names of the group/server to provide the service.
+- **Your Content**: To provide the service, we must store your messages and any content such as attachments and reactions.
+- **Legal Responsibility**: To ensure that users satisfy the legal age requirement to use the platform and to provide additional safety for minors, we must collect user's date of birth.
+- **Abuse Prevention**: To prevent our service from being attacked and protect your account, we collect your IP address, short device description for each session, and also temporarily store additional device information after initial registration.
```

Comment:
```suggestion
- **Abuse Prevention**: To prevent our service from being attacked and protect your account, we collect your IP address and a short device description for each session, and also temporarily store additional device information after initial registration.
```